aiida_pseudo/cli/install.py

```diff
@@ -140,3 +140,109 @@ def cmd_install_sssp(version, functional, protocol, traceback):
         family.set_cutoffs(cutoffs)
 
         echo.echo_success(f'installed `{label}` containing {family.count()} pseudo potentials')
+
+
+@cmd_install.command('pseudodojo')
+@options.VERSION(type=click.Choice(['03', '04']), default='04')
+@options.FUNCTIONAL(type=click.Choice(['pbe', 'pbesol', 'lda']), default='pbe')
+@options.REL(type=click.Choice(['sr', 'sr3plus', 'fr']), default='sr')
```

Comment:
Will this work if the version is 03? There should be only sr in that case I think.

Reply:
Yes, if you try to give an invalid combination of version / functional / rel / etc, you'll be given an error that it's an invalid configuration.